### PR TITLE
fix tooltip position

### PR DESCRIPTION
**Pull request by qiuminxu**
* Motivation for features / changes
The tooltip displayed when hovering on image of a pod is positioned suboptimally, this PR position the tooltip better. #2284 
* Technical description of changes

* Screenshots of UI changes
pod_viewer_new3

* Detailed steps to verify changes work correctly (as executed by you)
bazel run tensorboard:tensorboard -- --logdir=gs://cloud-tpu-tools
* Alternate designs / implementations considered

### CODE REVIEW

File: tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts [hunk 1]
```diff
@@ -356,8 +356,8 @@ Polymer({
 
             // Update the tooltip position and value
             d3.select(parent.$.tooltip)
-                .style('left', d3.event.pageX + 10 + 'px')
-                .style('top', d3.event.pageY - 10 + 'px')
+                .style('left', d3.select(this).attr('x') + 'px')
```
Comment (stephanwlee) [inline tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts]: Wouldn't you want some padding so it shows right (or bottom) of the current item?

Comment (qiuminxu) [reply tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts]: Done.

File: tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts [hunk 2]
```diff
@@ -356,8 +356,8 @@ Polymer({
 
             // Update the tooltip position and value
             d3.select(parent.$.tooltip)
-                .style('left', d3.event.pageX + 10 + 'px')
-                .style('top', d3.event.pageY - 10 + 'px')
+                .style('left', (~~d3.select(this).attr('x') + 50) + 'px')
```
Comment (stephanwlee) [inline tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts]: `~~` is a nifty trick but I had to think about it. How about just `Number(d3.select(this).attr('x'))`?

Comment (stephanwlee) [inline tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts]: Also I presume `50` is the width of the tooltip? Can we just measure the bounding client rect instead?

Comment (qiuminxu) [reply tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts]: Sure, done.

Comment (qiuminxu) [reply tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts]: The width of the tooltip is 200. I empirically adjusted the padding and find 50 works.

Comment (stephanwlee) [reply tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts]: I would advise against using such "magic" number. Is it at all impossible to calculate this value based on dimension somewhere?

Comment (qiuminxu) [reply tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts]: I would imagine d3.event.pageX and d3.event.pageY should just work or d3.select(this).attr('x') + rect.width + 5 and d3.select(this).attr('y') + rect.height + 5 should work. But neither works, except for these magic numbers.

**Pull request merged**